server/auth.js

```diff
@@ -19,12 +19,31 @@ if (!process.env.GOOGLE_APPLICATION_CREDENTIALS && process.env.NODE_ENV === 'dev
   process.env.GOOGLE_APPLICATION_CREDENTIALS = path.join(__dirname, '.auth.json')
 }
 
-// only public method, returns the authClient that can be used for making other requests
+// returns the authClient that can be used for making other requests
 exports.getAuth = async () => {
   if (authClient && process.env.NODE_ENV !== 'test') return authClient
   return setAuthClient()
 }
 
+// This is only used when data is fetched manually via a download link vs using the Google Drive library
+exports.getAccessToken = async () => {
+  const serviceAccountKey = process.env.GOOGLE_APPLICATION_JSON
+  const auth = new google.auth.GoogleAuth({
+    credentials: serviceAccountKey,
+    scopes: 'https://www.googleapis.com/auth/drive.readonly'
+  })
+
+  const accessToken = await auth.getClient()
+    .then((client) => {
+      return client.getAccessToken()
```

Comment:
Access token is valid for a certain period of time and can be used multiple times, so it is probably a bit unnecessary to get a new one every time. But I guess this works, too.

Reply:
I could use a refresh token based approach which would make sense but will take some more time. Shall we go with this first to get the library up and running again, and I can look into improving the access token handling later?

Reply:
Yes, it's probably not worth the effort to make this too complex. I guess the only real reason could be if Google had some usage limits on the number of access tokens that can be generated in timeframe X and this code would hit that limit. Which does not seem too likely, but like I said, I'm not an expert of Google APIs.

Reply:
> I'm not an expert of Google APIs
Same - I guess we'll notice once we hit the limit, if there is one.